universal-login-react/src/ui/TopUp/OnRamp/Safello.tsx

```diff
@@ -14,11 +19,12 @@ export const Safello = (props: SafelloProps) => {
   const url = getSafelloUrl(localizationConfig, safelloConfig, crypto, contractAddress);
   return (
     <iframe
+      className={`safello-iframe ${useThemeClassFor()}`}
```

Comment:
We should use `unilogin-component-safello0iframe`. To do that you can use `useClassFor('safello-iframe')`  😄